feature-libs/product/configurators/common/src/core/state/effects/index.ts

```diff
@@ -8,6 +8,6 @@ export const configuratorEffects: any[] = [
   ConfiguratorPlaceOrderHookEffects,
 ];
 
-export { ConfiguratorPlaceOrderHookEffects as ɵConfiguratorPlaceOrderHookEffects };
-export { ConfiguratorEffects as ɵConfiguratorEffects };
-export { ConfiguratorCartEffects as ɵConfiguratorCartEffects };
+export { ConfiguratorPlaceOrderHookEffects };
+export { ConfiguratorEffects };
+export { ConfiguratorCartEffects };
```

Comment:
It's an unwritten rule.... But unlike the modules, the ngrx _effects_ (and _reducers_) are what we agreed in the core team to keep private. It's to give us space a for functional development and improvements without breaking changes. 

(We've been there and accidentaly exported cart effects by mistake before 1.0 and we had a really hard time deprecating cart behaviors and fixing it without breaking changes.)

Please double-check that no ngrx effects or reducers can be imported in the app module from our libs (IDE should suggest only imports using an absolute path).